python_ta/checkers/global_variables_checker.py

```diff
@@ -38,10 +42,26 @@ def visit_name(self, node):
         """
         self._inspect_vars(node)
 
+    def visit_import(self, node):
+        """Save the names of imports, to prevent mistaking for global vars."""
+        self._store_name_or_alias(node)
+
+    def visit_importfrom(self, node):
+        """Save the names of imports, to prevent mistaking for global vars."""
+        self._store_name_or_alias(node)
+
+    def _store_name_or_alias(self, node):
+        if node.names[0][1] is not None:
```

Comment:
I'm not really sure about the structure of `node.names` (it would be good to update the documentation in the `nodes/` folder as well), but it seems like it's a list; this would suggest that a single `Import` node can actually have a bunch of imports (e.g., `import a, b, c`) and this method is only adding the first one.